.github/problem-matchers/generic.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What license is this under? Could you add a comment? Or does github require plain json and doesn't accept jsonc?

Reply:
@bjorn3  Thanks for the comment.  

 I didn't really reflect about the lincese. I'm assuming this may be under the project lincense. Am I wrong?
 
 As far as I know, I only saw json files, for the problem matcher. Nevertheless, it is possible to to add comments with a another label, like: "__comment_owner": "comment here" 
 
  e.g: https://github.com/ClangBuiltLinux/continuous-integration2/blob/main/.github/problem-matchers/generic.json
  
  Also, this commit is a proposal. For certain it can be fine tuned, or more problem matchers added.

Reply:
> I'm assuming this may be under the project license.

The kernel requires all source files to be annotated. Having said that, there are files that are handled differently (e.g. the GIF logo) and, in any case, the `.github` folder is not being submitted upstream, so it should be fine.